pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1408,7 +1439,12 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 	}
 
 	if vmi.Spec.Domain.CPU == nil || vmi.Spec.Domain.CPU.Model == "" {
-		domain.Spec.CPU.Mode = v1.CPUModeHostModel
+		// aarch64 not support host model well, set default CPU.Mode to host passthrough
+		if c.Architecture == "arm64" {
+			domain.Spec.CPU.Mode = v1.CPUModeHostPassthrough
+		} else {
+			domain.Spec.CPU.Mode = v1.CPUModeHostModel
+		}
```

Comment:
like the bios setting, i think this defaulting shoudl occur in the mutation webhook based on detecting `virt` machine type rather than here in the conversion.